package.json

```
@@ -92,12 +92,12 @@
     "@bbc/psammead-sitewide-links": "^0.1.4",
     "@bbc/psammead-styles": "^0.2.0",
     "@bbc/psammead-visually-hidden-text": "^0.1.7",
+    "@bbc/spartacus": "^0.0.9",
```

```suggestion
    "@bbc/spartacus": "^0.0.12",
```

This needs to be changed after the v0.0.12 is published to npm